README.md

```diff
@@ -1,41 +1,81 @@
-## Description
+# Mattermost for YunoHost
 
-A Yunohost package for [Mattermost](http://www.mattermost.org/), an open-source, self-hosted alternative to Slack.
+[![Integration level](https://dash.yunohost.org/integration/mattermost.svg)](https://dash.yunohost.org/appci/app/mattermost) ![](https://ci-apps.yunohost.org/ci/badges/mattermost.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/mattermost.maintain.svg)   
+[![Install Mattermost with YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=mattermost)
 
-[![Yunohost Build status](https://dash.yunohost.org/integration/mattermost.svg)](https://dash.yunohost.org/appci/app/mattermost)
+*[Lire ce readme en français.](./README_fr.md)*
 
-## Requirements
+> *This package allow you to install Mattermost quickly and simply on a YunoHost server.  
+If you don't have YunoHost, please see [here](https://yunohost.org/#/install) to know how to install and enjoy it.*
 
-Mattermost requires:
+## Overview
 
-* A x86_64 system (check with `uname -m`) (ARM Raspberry Pi won’t work, sorry),
-* Yunohost 3.0 or higher (check in Yunohost Admin panel),
-* MySQL 5.6 or higher, or MariaDB 10 or higher (check with `mysql --version`).
+Mattermost is a decentralized team communication software. For Slack enthusiasts, importing your data is possible, allowing a smooth transition to this free tool.
 
-## Installing
+**Shipped version:** 5.23.1
```

Comment:
I'm all for the new README template ; thanks for the PR.

However I definitely would like to limit the number to places to update when a new version gets released.

How about something that says :

```
Shipped version: [see app.src](link to app.src) (follows monthly Mattermost releases)
```

?

Reply:
It's your choice, but why make things more difficult for people and not comply with what everyone else is doing. It takes literally 2 seconds to change the version number...

Reply:
Yes. But I'd like to keep the app as close to Mattermost released version as possible (including several releases and dot-releases a month), and that involves making the release process as automated and painless as possible. So I'd really like to prevent new steps to be added on each upgrade, because small things add up.